src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,48 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int AMOUNT_OF_ARRAYS = 2;
+    private static final int AMOUNT_OF_ITEMS = 10;
+    private static final int INDEX_OF_KEYS = 0;
+    private static final int INDEX_OF_VALUES = 1;
+    private Object[][] dictionary = new Object[AMOUNT_OF_ARRAYS][AMOUNT_OF_ITEMS];
+    private int size = 0;
```

Comment:
All non-constant fields should be initialized in the constructor (from common mistakes:)
``
```suggestion
    private Object[][] dictionary;
    private int size;
    
    public StorageImpl() {
         dictionary = new Object[AMOUNT_OF_ARRAYS][AMOUNT_OF_ITEMS];
         size = 0;
    }
```